packages/vscode-extension/src/commands/ManageAccounts.ts

```diff
@@ -11,7 +11,7 @@ import { MANAGE_ACCOUNTS, DISMISS } from '../service/TranslationKeys';
 import * as fs from 'fs';
 import * as path from 'path';
 
-const COMMAND_NAME = 'manageaccounts';
+const COMMAND_NAME = 'setupaccount';
```

Comment:
I think we should rename the file accordingly.

Reply:
Renamed.